plasmapy/particles/tests/test_parsing.py

```diff
@@ -331,16 +331,17 @@ def test_parse_InvalidParticleErrors(arg, kwargs):
         )
 
 
-@pytest.mark.parametrize("arg", particle_zoo.everything - {"p+"})
-def test_parse_InvalidElementErrors(arg):
+def test_parse_InvalidElementErrors(particle):
     r"""Tests that _parse_and_check_atomic_input raises an
     InvalidElementError when the input corresponds to a valid
     particle but not a valid element, isotope, or ion."""
+    if particle == Particle("p+"):
+        return  # TODO verify this case should still be a skip
```

Comment:
```suggestion
    if particle == Particle("p+"):
        return
```
Yep, protons should still be skipped here.  The reason is that protons are the only particles that count as both a special particle and an element, so this test doesn't apply to them.